src/brogue/IncludeGlobals.h

```diff
@@ -45,6 +45,9 @@ extern short **allySafetyMap;
 extern short **chokeMap;
 
 extern const short nbDirs[8][2];
+static inline pos neighborInDir(pos p, short d) {
```

Comment:
perhaps add a comment describing the purpose of the function. it wasn't immediately clear to me from the name.

Reply:
I think a comment in Globals.h is less useful than a clearer name - getNeighborPosInDir, neighborPosInDir, posOneSquareAwayInDir?

Reply:
After a bunch of waffling back and forth I've renamed it to `posNeighborInDirection`. I think this is the best balance of absolutely clear and still reasonably short (in particular, the expectation is that this helper function is used as a primitive around which other coordinate-manipulating code is built; so it should be easy to scan so that those other functions which call it are more readable).

Leading with `pos__` makes it clearly position-related, so if you're unfamiliar with it its purpose is more obvious (or at least guessable).

I've also added a comment before the function which would be visible in e.g. intellisense to give even more context to someone looking through the code.

Reply:
The comment is helpful. My suggestion for the name is getNeighboringPosition.

Reply:
Let's merge as is, if we decide the new name is really a massive problem later we can follow up. I checked the revised branch and it's fine on regression.